app/src/main/java/org/fossasia/openevent/utils/SpeakerIntent.java

```diff
@@ -41,9 +41,9 @@ public void onClick(View view) {
                     Toast.makeText(view.getContext(), "What did you click?? :P", Toast.LENGTH_SHORT).show();
                 }
 
-                if (url.equals(null)) {
+                if (url.equals(null) || url.isEmpty()) {
```

Comment:
@suheb You just need to check for null like `url != null` and it will work. No need for do this.